lib/geojson_wrapper.ts

```diff
@@ -1,8 +1,7 @@
 import Point from '@mapbox/point-geometry';
-import {VectorTileFeature, VectorTileLayer, type VectorTile} from '@mapbox/vector-tile';
 import type {TileFeature, AnyProps} from 'supercluster';
 import {type Feature as GeoJSONVTFeature, Geometry} from 'geojson-vt';
-import Pbf from 'pbf';
+import type {VectorTileFeatureLike, VectorTileLayerLike} from '../index';
```

Comment:
I think this creates circular dependency of some sort, it might be better to place these types in a different file that is not index.ts and re-export them.